Doc/whatsnew/3.11.rst

```diff
@@ -126,6 +126,10 @@ Deprecated
 Removed
 =======
 
+* The :func:`asyncio.coroutine` :term:`decorator` enabling legacy
```

Comment:
```suggestion
* The :func:`@asyncio.coroutine <asyncio.coroutine>` :term:`decorator` enabling legacy
```

Reply:
Thanks, I updated this together with another suggestion